### ISSUE TITLE
Feature Request: axis scales are correct regardless of orientation

### ISSUE THREAD

**Issue by robertlugg** [openscad/openscad#3362]
Depending on the view, the numbers on the primary axes are flipped.

This is what I consider to be a good case:
![axis_view_good](https://user-images.githubusercontent.com/6054540/85953156-33117f00-b923-11ea-8fb1-13fa21660c7a.png)

The next two are the bad cases.  As the view changes the numbers are shown upside down.

Note that 20 is shown as 02 in one of the axis:
![axis_view_bad2](https://user-images.githubusercontent.com/6054540/85953155-3278e880-b923-11ea-88b7-95b03ecbc112.png)

Note that 20s are shown as 50s:
![axis_view_bad1](https://user-images.githubusercontent.com/6054540/85953157-33117f00-b923-11ea-848d-0959e6666fe2.png)

I actually can't say if this is wrong or how other tools work, but I'm wondering if the numbers somehow adjust so that they are always front facing and flipped to be correct from the viewport.

**Comment by haarp** [comment]
Yes, the axes are often not very readable. On top of them becoming rotated upside down, they can also be rotated to become perpendicular to the viewport, rendering them unreadable. Plus, the numbers are often hard to read due to lack of contrast.

The same goes for the X Y Z marker.

I would propose that the numbers always stay parallel to the viewport. In addition, the numbers (and probably the axes themselves) should dynamically adjust their color. A clever way of doing so is to always show the inversion of whatever color is behind them.

**Comment by jordanbrown0** [comment]
Note that current development snapshots use a much nicer font.  While the labels can still be up-side down or reversed, they aren't nearly as visually ambiguous.
